cmd/release/command.go

```diff
@@ -0,0 +1,56 @@
+package release
+
+import (
+	"io"
+	"os"
+
+	"github.com/giantswarm/microerror"
+	"github.com/giantswarm/micrologger"
+	"github.com/spf13/cobra"
+)
+
```

Comment:
Problem 1: With that we'll lose ability to review releases before merge.

Reply:
mmm... what's need to be reviewed? I was assuming at this point everything worked fine and the only missing part was creating the tag.

Reply:
E.g:

- I don't want this release yet because I want to merge something.
- I don't want this release because there is a bug.
- We do this release tomorrow because of something.

Reply:
Then don't execute this command until

- You merge something missing
- You fixed the bug
- Wait until tomorrow
?

I mean, this command is not automatically executed during a workflow. It's manually executed, each team decides when.

Reply:
We need more feedback, good people at @giantswarm/sig-releng

Reply:
The biggest problem I see here is that Jose did something cool for hackathon and now got a downer with all the cautious hesitance. Not saying the good people here caring and holding the line is a bad thing but we should maybe go for it and solve the real problems we have. I do not see a new problem here. All the arguments for not doing it are already true with the manual process. When you still disagree please get together and talk directly in a timeboxed way. Let's make Jose run again. 🚀

Reply:
Let's not create drama here.

I will elaborate a bit because I see it isn't obvious.

> hen don't execute this command until
> 
> * You merge something missing
> * You fixed the bug
> * Wait until tomorrow

This is with the assumption you have all that knowledge. If you bypass creating a PR how can you can share you're about to create a release and learn if others don't have anything to say?

I understand where you're coming from. You're a team that pretty much does azure-operator and you can communicate everything internally. But we need to have a way that works for everything else in the company.

This goes pretty much in the direction opposite to https://github.com/giantswarm/giantswarm/issues/9251. With pushing directly to master basically all releases sneak in. No PRs are created so we have zero validation from the CI.

In general pushing directly to master from developer machines is something I'd like to avoid. This sounds completely wrong to me. Ideally that should be blocked on GitHub level. 

Maybe splitting this into two commands that create a PRs for release + create GitHub release and create a PR bumping version is an idea.

Having said that I will disagree and commit that if people like it.